### ISSUE TITLE
Poor contrast

### ISSUE THREAD

**Issue by edent** [nothingrandom/barebones#6]
The combination of thin font and low contrast make this text extremely hard to read.

![readability](https://cloud.githubusercontent.com/assets/837136/11030019/1b70f61a-86c3-11e5-8860-ceefca055fb3.png)
# AAA on #FFF does not meet WCAG guidelines.  https://leaverou.github.io/contrast-ratio/#%23AAA-on-%23FFF

**Comment by nothingrandom** [comment]
Thank you for pointing this out. I've made the text #757575 now, which has a score of 4.6.
